rasa_sdk/tracing/instrumentation/instrumentation.py

```diff
@@ -99,14 +99,23 @@ def traceable(
 
     @functools.wraps(fn)
     def wrapper(self: T, *args: Any, **kwargs: Any) -> S:
-        attrs = (
-            attr_extractor(self, *args, **kwargs)
-            if attr_extractor and should_extract_args
-            else {}
-        )
+        if fn.__name__ == "_create_api_response":
+            attrs = (
```

Comment:
Let's add a comment to say this requires a separate branch because _create_api_response is a static method, I'd also add `and if isinstance(self, ActionExecutor)` to make it explicit this only refers to that method.